source/devices/AM65X/linux/Release_Specific_Migration_Guide.rst

```diff
@@ -1,3 +1,36 @@
 .. _release-specific-migration-guide:
 
-.. include:: /linux/Release_Specific/_CoreSDK_Linux_Migration_Guide.rst
```

Comment:
Since you're dropping the only other include for this file, it's probably worth moving `source/linux/Release_Specific/_CoreSDK_Linux_Migration_Guide.rst` to `source/linux/Release_Specific_CoreSDK_Migration_Guide.rst` outright.

Reply:
fixed in the update. please review